syft/workers/base.py

```diff
@@ -986,6 +979,13 @@ def force_detail(worker: AbstractWorker, worker_tuple: tuple) -> tuple:
 
         return result
 
+    @staticmethod
+    def register_message_handlers():
+        if sy.dependency_check.crypten_available:
+            from syft.frameworks.crypten.message_handler import CryptenMessageHandler
+
+            BaseWorker._framework_message_handler["crypten"] = CryptenMessageHandler
+
```

Comment:
No need for a function to do that, that attribute could be initialized at first, and message handlers would be appended only if the dependency check passes, which is something the message hander itself checks for (with a static function `MessageHandler.is_supported()` for example)

Reply:
Hmmm...or couldn't we add this ```registry mechanism``` directly in the ```CryptenMessageHandler``` class file (if we import that file it means we have the dependencies hence we should add it to the ```_framework_message_handlers```?)

@karlhigley what do you think?